server/index.js

```diff
@@ -43,39 +55,54 @@ app.get('/get_package_info', (req, res) =>{
 
 
 /* ***** add_package  *****
-    desc: add a package to start tracking
+    desc: add a package to start tracking. 3 step mechanism
+      1. Validate user 
+      2. Add package to start tracking in PKGE
+      3. Add package reference to user in Firestore
     inputs: 
       trackingNumber: tracking number of the package to track
     outputs: 
       code 200: successfully adds the new package
       code 400: unable to add package, more error codes and error messages in the return response
 */ 
 app.post('/add_package', (req, res) => {
```

Comment:
Is there a reason for using post /add_package and get /get_packages instead of post /package and get /package?

Reply:
No, we should use what you suggest.